setup/sql/dbupdate_05.php

```diff
@@ -74,4 +74,12 @@
 <#5437>
 <?php
 $ilCtrlStructureReader->getStructure();
-?>
\ No newline at end of file
+?>
+<?php
```

Comment:
This looks like an issue caused by a merge.

Reply:
Thx, i think this was my fault.